src/ui/style/palette.scss

```diff
@@ -108,3 +108,13 @@ $tab-inactive-background-dark: rgba(255, 255, 255, 0.06);
 $tab-inactive-text-light: rgba(0, 64, 115, 0.6);
 $tab-inactive-text-dark: rgba(255, 255, 255, 0.6);
 $tab-active-background: #004073;
+
+//file drag and drop
+$file-picker-background-light: #f2f6f9;
+$file-picker-background-dark: #242433;
+$file-picker-background-focused-light: #d5e5ed;
+$file-picker-background-focused-dark: #202c3f;
+$file-picker-border-light: hsla(207, 100%, 23%, 0.4);
+$file-picker-border-dark: hsl(0, 0%, 100%);
```

Comment:
hsl for black color seems a bit overkill 😄

Reply:
For some reason other values such as hexadecimal does not work